wremnants/include/EtaPtCorrelatedEfficiency.h

```diff
@@ -0,0 +1,304 @@
+#ifndef WREMNANTS_EtaPtCorrelatedEfficiency_h
+#define WREMNANTS_EtaPtCorrelatedEfficiency_h
+
+#include "TROOT.h"
+#include "TH2D.h"
+#include "TH3D.h"
+#include "TFile.h"
+#include "TMath.h"
+
+#include <eigen3/Eigen/Dense>
+#include <eigen3/Eigen/Eigenvalues>
+//#include <stdlib.h>
+//#include <stdio.h>
+#include <cstdlib> 
+#include <cstdio>
+#include <cmath>
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <cassert>
+
+namespace wrem {
+
+    // ================================================
+    // Some functions to be used for EtaPtCorrelatedEfficiency
+    // ================================================
+    // TODO:
+    // put functions in another header file
+    // write a base class to make all functions derive from, at least for polynomials
+    // write a generic class for polynomials
+    class pol3_custom {
+    public:
+        pol3_custom() {};
+        pol3_custom(const double& xMin, const double& xRange) {
+            xMinNorm_ = xMin;
+            xRangeNorm_ = xRange;
+        };
+        double operator() (double *x, double *p) {
```

Comment:
Why pointers?

Reply:
The function I copied was implemented like that, anyway this header is not obsolete, we are using a simpler numpy implementation. It could be removed but I'd actually keep it in case it turns out to be useful again